index.d.ts

```diff
@@ -534,6 +533,60 @@ declare module 'plaid' {
     ytd_principal_paid: number | null;
   }
 
+  interface CreditLiability {
+    account_id: string | null;
+    aprs: AnnualPercentageRate[];
+    is_overdue: boolean | null;
+    last_payment_amount: number;
+    last_payment_date: string;
+    last_statement_issue_date: string;
```

Comment:
I'd suggest using `Iso8601DateString | null` for the type here.

Reply:
Thanks for the suggestion. Can you point to where this is used Typescript spec?